exercises/pokemon_server.livemd

```diff
@@ -117,7 +117,7 @@ Pokemon health should not fall below 0.
 
 ```elixir
 {:ok, charmander} = Pokemon.new(%{name: "charmander", type: "fire", damage: 110})
```

Comment:
I think there is an inconsistency in that I don't see `damage` mentioned at the top of the exercise. I do see `attack`. Maybe this should be fixed as well?

I also see a curious line where it appears the exercise wants us to make a `Pokemon.attack/2` function, but I see inconsistent return values, sometimes being the first monster and sometimes the second. Not being familiar with the work around this exercise I might be mistaken but just sharing as I read this over when seeing the PR.

Reply:
Yup, they should both be `attack`. That's a mistake on my part.